src/main/java/org/sonarlint/intellij/analysis/SonarLintAnalysisConfigurator.java

```diff
@@ -42,10 +42,8 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 import org.sonarlint.intellij.ui.SonarLintConsole;
-
-import static org.sonarsource.sonarlint.core.AnalysisConfiguration.InputFile;
-
-import org.sonarsource.sonarlint.core.IssueListener;
+import org.sonarsource.sonarlint.core.client.api.IssueListener;
+import org.sonarsource.sonarlint.core.client.api.ClientInputFile;
 
 import java.nio.charset.Charset;
 import java.nio.file.Path;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.nio.file.Path'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)